pkgs/applications/misc/mupdf/default.nix

```diff
@@ -45,36 +44,23 @@
 
 assert enablePython -> enableCxx;
 
-let
-
-  freeglut-mupdf = freeglut.overrideAttrs (old: rec {
-    pname = "freeglut-mupdf";
-    version = "3.0.0-r${src.rev}";
-    src = fetchFromGitHub {
-      owner = "ArtifexSoftware";
-      repo = "thirdparty-freeglut";
-      rev = "13ae6aa2c2f9a7b4266fc2e6116c876237f40477";
-      hash = "sha256-0fuE0lm9rlAaok2Qe0V1uUrgP4AjMWgp3eTbw8G6PMM=";
-    };
-  });
-
-in
 stdenv.mkDerivation rec {
-  version = "1.23.6";
+  version = "1.24.0";
   pname = "mupdf";
 
   src = fetchurl {
-    url = "https://mupdf.com/downloads/archive/${pname}-${version}-source.tar.gz";
-    sha256 = "sha256-rBHrhZ3UBEiOUVPNyWUbtDQeW6r007Pyfir8gvmq3Ck=";
+    url = "https://mupdf.com/downloads/archive/mupdf-${version}-source.tar.gz";
+    hash = "sha256-UvYwA6b02J8jTp7ftLTIPSFrg6rrMjz9pgR8t1RZmuA=";
   };
 
-  patches = [ ./0001-Use-command-v-in-favor-of-which.patch
-              ./0002-Add-Darwin-deps.patch
-              ./0003-Fix-cpp-build.patch
-            ];
+  patches = [
+    ./0002-Add-Darwin-deps.patch
+    ./0003-Fix-cpp-build.patch
+  ];
 
   postPatch = ''
-    substituteInPlace Makerules --replace "(shell pkg-config" "(shell $PKG_CONFIG"
+    substituteInPlace Makerules \
+      --replace-fail "(shell pkg-config" "(shell $PKG_CONFIG"
```

Comment:
If you aren't going to set `PKG_CONFIG` below in the `makeFlags`, then it should be removed here too (also `buildPackages` is now an unused argument)

I forget, but is it needed for cross to build, though?

Reply:
> I forget, but is it needed for cross to build, though?

Yes, you have to use `$PKG_CONFIG` for cross builds. I tested that they still work.

Reply:
Oh they still work without setting the `makeFlag`?

That's fine then, you can mark this as resolved if cross still works